docs/docs-content/architecture/architecture-overview.md

```diff
@@ -87,3 +87,29 @@ be optionally downloaded from a self-hosted private repository instead of pullin
 repository.
 
 ![Self-hosted Palette architecture diagram](/architecture_architecture-on-prem-detailed.webp)
+
+### Message Brokers
+
+Palette requires reliable, scalable and secure communication. The internal microservices use a Publish-Subscribe pattern
+implemented with [gRPC](https://grpc.io/) to achieve this. In order to support this communication pattern, a message
+broker service acts as the central hub for message exchange. Palette message brokers are automatically scaled, ensuring
+that a quorum is available for each cluster. The broker system is designed to provide the following functionality.
```

Comment:
It's unclear which "cluster" we are referring to here. AFAIK, message brokers are only running in the Palette management plane.

Reply:
Clarified all cluster instances now. Thanks so much!